src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/MoveService.kt

```diff
@@ -23,11 +23,11 @@ class MoveService(
     return maybeMove?.let {
       val moveEvents = eventRepository.findAllByEventableId(it.moveId)
       val journeyId2Events =
-        eventRepository.findByEventableIdIn(it.journeys.map { it.journeyId }).groupBy { it.eventableId }
+        eventRepository.findByEventableIdIn(it.journeys.map { j -> j.journeyId }).groupBy { e -> e.eventableId }
 
       val journeysWithEvents = it.journeys.map { journey ->
         journey.copy(events = journeyId2Events[journey.journeyId] ?: listOf())
-      }
+      }.sortedBy { journey -> journey.pickUpDateTime }
```

Comment:
This is where the journeys for moves are sorted for ready for the presentation layer.